### ISSUE TITLE
Mousecape pack gallery

### ISSUE THREAD

**Issue by s4Dt0y** [alexzielenski/Mousecape#232]
I'm new to this app, and have not been able to find too many capes...it would be nice if there was a single place newcomers could go to browse capes.

**Comment by exninja** [comment]
Here's a starter pack I found: [https://www.deviantart.com/allannyholm/art/Mousecape-Throwback-2021-Mac-cursors-781872368](https://www.deviantart.com/allannyholm/art/Mousecape-Throwback-2021-Mac-cursors-781872368)

**Comment by s4Dt0y** [comment]
Yes, I have been using those, but I was thinking something like a mouse cape pages website or a discussion, or a wiki.

**Comment by PK761** [comment]
thanks for the starter pack
